WordPress/Classes/Utility/Media/ImageDownloader+Gravatar.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+extension ImageDownloader {
+
+    func downloadGravatarImage(with email: String, completion: @escaping (UIImage?) -> Void) {
+
+        guard let url = gravatarUrl(for: email) else {
+            completion(nil)
+            return
+        }
+
+        downloadImage(at: url) { image, _ in
+            DispatchQueue.main.async {
+
+                guard let image else {
+                    completion(nil)
+                    return
+                }
+
+                completion(image)
+            }
+        }
+    }
+
+    private func gravatarUrl(for email: String) -> URL? {
```

Comment:
There should be some existing code for creating the URLs, potentially this: `downloadGravatarWithEmail`.

Reply:
As discussed, I've moved over this logic to `Gravatar.swift` 🙇‍♀️ 
https://github.com/wordpress-mobile/WordPressUI-iOS/pull/133